src/features/filter.ts

```diff
@@ -1,10 +1,63 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+type SetQueryAction = {
+  type: 'filter/QUERY_SET',
+  payload: string,
 };
 
+type ClearQueryAction = {
+  type: 'filter/QUERY_CLEAR',
+};
+
+type SetStatusAction = {
+  type: 'filter/STATUS_SET',
+  payload: string,
+};
+
+const setQuery = (query: string): SetQueryAction => ({
+  type: 'filter/QUERY_SET',
+  payload: query,
+});
+
+const clearQuery = (): ClearQueryAction => ({
+  type: 'filter/QUERY_CLEAR',
+});
+
+const setStatus = (status: string): SetStatusAction => ({
```

Comment:
The `status` parameter of the `setStatus` function should use `Status` instead of `string` type.
Fix all cases